scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -274,22 +272,22 @@ sealed trait TypedPipe[+T] extends Serializable {
    *   collect { case Some(x) => fn(x) }
    * }
    */
-  def collect[U](fn: PartialFunction[T, U]): TypedPipe[U] =
+  def collect[U](fn: PartialFunction[T, U])(implicit m: Quoted): TypedPipe[U] =
     filter(PartialFunctionToFilter(fn)).map(fn)
 
   /**
    * Attach a ValuePipe to each element this TypedPipe
    */
-  def cross[V](p: ValuePipe[V]): TypedPipe[(T, V)] =
-    TypedPipe.CrossValue(this, p).withLine
+  def cross[V](p: ValuePipe[V])(implicit m: Quoted): TypedPipe[(T, V)] =
+    TypedPipe.CrossValue(this, p).withQuoted
 
   /** prints the current pipe to stdout */
-  def debug: TypedPipe[T] =
-    TypedPipe.DebugPipe(this).withLine
+  def debug(implicit m: Quoted): TypedPipe[T] =
+    TypedPipe.DebugPipe(this).withQuoted
 
   /** adds a description to the pipe */
   def withDescription(description: String): TypedPipe[T] =
```

Comment:
why don't we want `Quoted` here. Can you add the comment why? Is it the assumption that users get full control of the note, and there is no need to add the line/file info that Quoted can give?

Reply:
just added updated the scaladoc